pkg/storage/bucket/client_test.go

```diff
@@ -127,3 +129,99 @@ func TestClientMock_MockGet(t *testing.T) {
 
 	wg.Wait()
 }
+
+func TestClient_ConfigValidation(t *testing.T) {
+	testCases := []struct {
+		name           string
+		cfg            Config
+		expectingError bool
```

Comment:
Nit: it would be good to check for specific errors to make sure it works as expected.